Library/test-helpers/lib.sh

```diff
@@ -1971,86 +1930,52 @@ limeconPrepareRegistrarImage() {
 true <<'=cut'
 =pod
 
-=head2 limeconPrepareVerifierImage
+=head2 limeconRun
 
-Prepare podman image with keylime verifier. Install keylime verifier.
-
-    limeconPrepareAgentImage [-f|--file DOCKER_FILE] TAG
+Container run via podman with specified parameters.
 
-=over
+    limeconRun NAME TAG IP NETWORK CMD_RUN CMD_SETUP 
 
-=item -f, --file DOCKER_FILE
+=item NAME
 
-Parameter specify use of non-default docker file.
+Set name of container.
 
 =item TAG
 
 Name of image tag.
 
-=back
-
-Returns 0.
-
-=cut
-
-limeconPrepareVerifierImage() {
-
-    if [ "$1" == "-f" -o "$1" == "--file" ]; then
-        local DOCKER_FILE=$2
-        shift 2
-    else
-        echo "Using default docker file: ${limeLibraryDir}/Dockerfile.verifier"
-        DOCKER_FILE="${limeLibraryDir}/Dockerfile.verifier"
-    fi
-
-    local TAG=$1
-
-    ls /root/.ssh/id_*.pub || ssh-keygen -t rsa -N "" -f /root/.ssh/id_rsa
-    cp /root/.ssh/id_*.pub .
-    cp ${limeLibraryDir}/lime_con_start.sh .
-    podman build -t=$TAG --file $DOCKER_FILE .
-}
-
-true <<'=cut'
-=pod
+=item IP
 
-=head2 limeconPrepareSystemdImage
+IP address of container.
 
-Prepare podman image with installed and running.
+=item NETWORK
 
-    limeconPrepareSystemdImage [-f|--file DOCKER_FILE] TAG
+Name of used podman network.
 
-=over
+=item CMD_RUN
 
-=item -f, --file DOCKER_FILE
+Specify running command at start of container.
 
-Parameter specify use of non-default docker file.
+=item CMD_SETUP
```

Comment:
Could we rather use something like `EXTRA_PODMAN_ARGS`? The content is not an actual command, just arguments.